test/unit/gateways/moneris_us_test.rb

```diff
@@ -22,6 +22,21 @@ def test_default_options
     assert_equal "qatoken", @gateway.options[:password]
   end
 
+  def test_successful_verify
+    @gateway.expects(:ssl_post).twice.returns(successful_purchase_response)
```

Comment:
Even though there'd be duplication, I'd still go for having a `successful_authorization_response` if for nothing else than to indicate intent.  When I look at the `test_successful_verify` method, my first thought is that there must something amiss since it's stubbing out the purchase call twice rather than stubbing out the `authorize` and the `capture` for $0.  

I'm also in the camp that there's value in having a unit test for `test_successful_verify_with_failed_void` to ensure that `ignore_result` is hooked up appropriately.

I'll mess with it here for a few minutes and see what I come up with.